stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -508,6 +515,9 @@ internal class ChatDomainImpl internal constructor(
                     this
                 )
             activeChannelMapImpl[cid] = channelRepo
+            GlobalScope.launch(Dispatchers.Main) {
```

Comment:
- Why do we need to go to the main thread here?
- If we do need this, please use `DispatcherProvider` instead

Reply:
If addSource it called in another thread different than the main one, it crashes, so this is needed to move to the main thread.

Reply:
I think GlobalScope usage is not the best practice to launch a coroutine. ChatDomain contains its' own scope, you can use it here.

Reply:
Changed for `DispatcherProvider`

Reply:
Sorry @ogkuzmin, I didn't see you message. I'll change the scope.